src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) throws IOException {
+        return Files.readAllLines(Paths.get(filePath));
```

Comment:
Avoid hardcoding the file path in the service implementation. The file path should be passed as a parameter to the method, which is correctly done here. However, make sure that the file path used in the method call is not hardcoded elsewhere in the application.